apps/chat/src/store/application/application.reducers.ts

```diff
@@ -65,6 +65,9 @@ export const applicationSlice = createSlice({
     ) => {
       state.loading = true;
     },
+    updateSuccess: (state) => {
+      state.loading = false;
```

Comment:
loading should be reset on `editSuccess` and it looks like we don't need `updateSuccess`